Check-Style fails for Google-Style for RightCurlyCheck

## Describe the bug

Using google_check.xml for Java check-style checking with Google Style fails.
The issue is now fixed with underlying check-style system in version 10.9.0

```
com.puppycrawl.tools.checkstyle.api.CheckstyleException: cannot initialize module TreeWalker - Token "LITERAL_SWITCH" was not found in Acceptable tokens list in check com.puppycrawl.tools.checkstyle.checks.blocks.RightCurlyCheck
        at com.puppycrawl.tools.checkstyle.Checker.setupChild(Checker.java:481)
        at com.puppycrawl.tools.checkstyle.api.AutomaticBean.configure(AutomaticBean.java:201)
        at com.puppycrawl.tools.checkstyle.Main.runCheckstyle(Main.java:412)
        at com.puppycrawl.tools.checkstyle.Main.runCli(Main.java:339)
        at com.puppycrawl.tools.checkstyle.Main.execute(Main.java:196)
        at com.puppycrawl.tools.checkstyle.Main.main(Main.java:131)
Caused by: com.puppycrawl.tools.checkstyle.api.CheckstyleException: Token "LITERAL_SWITCH" was not found in Acceptable tokens list in check com.puppycrawl.tools.checkstyle.checks.blocks.RightCurlyCheck
        at com.puppycrawl.tools.checkstyle.TreeWalker.registerCheck(TreeWalker.java:224)
        at com.puppycrawl.tools.checkstyle.TreeWalker.setupChild(TreeWalker.java:134)
        at com.puppycrawl.tools.checkstyle.api.AutomaticBean.configure(AutomaticBean.java:201)
        at com.puppycrawl.tools.checkstyle.Checker.setupChild(Checker.java:476)
        ... 5 more
Checkstyle ends with 1 errors.
```

## Expected behavior

CheckStyle should not fail.

The checkstyle and `google-java-format` versions are not controlled through dependabot and should be.

## Steps to Reproduce

docker run \
  -e RUN_LOCAL=true \
  -e ACTIONS_RUNNER_DEBUG=true \
  -e DISABLE_ERRORS=false \
  -e ERROR_ON_MISSING_EXEC_BIT=true \
  -e LINTER_RULES_PATH=. \
  -e MULTI_STATUS=false \
  -e VALIDATE_ALL_CODEBASE=true \
  -e VALIDATE_JAVA=true \
  -e JAVA_FILE_NAME=google_checks.xml \
  -v $(pwd):/tmp/lint \
  ghcr.io/super-linter/super-linter:v3.13.5

## Additional context

The check-style version is fixed in [Dockerfile](https://github.com/super-linter/super-linter/blob/b335a00df3cef725672b11abba8fa9895f7f06a5/Dockerfile#L37)

--- a/.github/dependabot.yml
+++ b/.github/dependabot.yml
@@ -37,3 +37,15 @@ updates:
     schedule:
       interval: "weekly"
     open-pull-requests-limit: 10
+
+  # Maintain dependencies for Java
+  - package-ecosystem: "gradle"
+    directory: "/dependencies/checkstyle"
+    schedule:
+      interval: "weekly"
+    open-pull-requests-limit: 10
+  - package-ecosystem: "gradle"
+    directory: "/dependencies/google-java-format"
+    schedule:
+      interval: "weekly"
+    open-pull-requests-limit: 10

--- a/Dockerfile
+++ b/Dockerfile
@@ -34,12 +34,10 @@ FROM python:3.11.5-alpine3.17 as base_image
 ################################
 # Set ARG values used in Build #
 ################################
-ARG CHECKSTYLE_VERSION='10.3.4'
 ARG CLJ_KONDO_VERSION='2023.05.18'
 # Dart Linter
 ## stable dart sdk: https://dart.dev/get-dart#release-channels
 ARG DART_VERSION='2.8.4'
-ARG GOOGLE_JAVA_FORMAT_VERSION='1.18.1'
 ## install alpine-pkg-glibc (glibc compatibility layer package for Alpine Linux)
 ARG GLIBC_VERSION='2.34-r0'
 ARG KTLINT_VERSION='0.47.1'

--- a/Dockerfile
+++ b/Dockerfile
@@ -92,7 +92,7 @@ RUN apk add --no-cache \
 ########################################
 # Copy dependencies files to container #
 ########################################
-COPY dependencies/* /
+COPY dependencies/ /
 
 ###################################################################
 # Install Dependencies                                            #

--- a/dependencies/checkstyle/build.gradle
+++ b/dependencies/checkstyle/build.gradle
@@ -1,0 +1,12 @@
+repositories {
+  mavenLocal()
+  mavenCentral()
+}
+
+// Hold this dependency here so we can get automated updates using DependaBot
+dependencies {
+  implementation 'com.puppycrawl.tools:checkstyle:10.3.4'
+}
+
+group 'com.github.super-linter'
+version '1.0.0-SNAPSHOT'

--- a/dependencies/google-java-format/build.gradle
+++ b/dependencies/google-java-format/build.gradle
@@ -1,0 +1,12 @@
+repositories {
+  mavenLocal()
+  mavenCentral()
+}
+
+// Hold this dependency here so we can get automated updates using DependaBot
+dependencies {
+  implementation 'com.google.googlejavaformat:google-java-format:1.18.1'
+}
+
+group 'com.github.super-linter'
+version '1.0.0-SNAPSHOT'

--- a/scripts/install-checkstyle.sh
+++ b/scripts/install-checkstyle.sh
@@ -2,6 +2,9 @@
 
 set -euo pipefail
 
+CHECKSTYLE_VERSION="$(grep <"/dependencies/checkstyle/build.gradle" "checkstyle" | awk -F ':' '{print $2}')"
+echo "Installing Checkstyle: ${CHECKSTYLE_VERSION}"
+
 url=$(curl -s \
   -H "Accept: application/vnd.github+json" \
   -H "Authorization: Bearer $(cat /run/secrets/GITHUB_TOKEN)" \

--- a/scripts/install-checkstyle.sh
+++ b/scripts/install-checkstyle.sh
@@ -2,7 +2,7 @@
 
 set -euo pipefail
 
-CHECKSTYLE_VERSION="$(grep <"/dependencies/checkstyle/build.gradle" "checkstyle" | awk -F ':' '{print $2}')"
+CHECKSTYLE_VERSION="$(grep <"dependencies/checkstyle/build.gradle" "checkstyle" | awk -F ':' '{print $3}' | tr -d "'")"
 echo "Installing Checkstyle: ${CHECKSTYLE_VERSION}"
 
 url=$(curl -s \

--- a/scripts/install-checkstyle.sh
+++ b/scripts/install-checkstyle.sh
@@ -2,7 +2,7 @@
 
 set -euo pipefail
 
-CHECKSTYLE_VERSION="$(grep <"dependencies/checkstyle/build.gradle" "checkstyle" | awk -F ':' '{print $3}' | tr -d "'")"
+CHECKSTYLE_VERSION="$(grep <"checkstyle/build.gradle" "checkstyle" | awk -F ':' '{print $3}' | tr -d "'")"
 echo "Installing Checkstyle: ${CHECKSTYLE_VERSION}"
 
 url=$(curl -s \

--- a/scripts/install-google-java-format.sh
+++ b/scripts/install-google-java-format.sh
@@ -2,7 +2,7 @@
 
 set -euo pipefail
 
-GOOGLE_JAVA_FORMAT_VERSION="$(grep <"dependencies/google-java-format/build.gradle" "google-java-format" | awk -F ':' '{print $2}')"
+GOOGLE_JAVA_FORMAT_VERSION="$(grep <"dependencies/google-java-format/build.gradle" "google-java-format" | awk -F ':' '{print $3}' | tr -d "'")"
 echo "Installing Google Java Format: ${GOOGLE_JAVA_FORMAT_VERSION}"
 
 url=$(curl -s \

--- a/scripts/install-google-java-format.sh
+++ b/scripts/install-google-java-format.sh
@@ -2,7 +2,7 @@
 
 set -euo pipefail
 
-GOOGLE_JAVA_FORMAT_VERSION="$(grep <"dependencies/google-java-format/build.gradle" "google-java-format" | awk -F ':' '{print $3}' | tr -d "'")"
+GOOGLE_JAVA_FORMAT_VERSION="$(grep <"google-java-format/build.gradle" "google-java-format" | awk -F ':' '{print $3}' | tr -d "'")"
 echo "Installing Google Java Format: ${GOOGLE_JAVA_FORMAT_VERSION}"
 
 url=$(curl -s \

--- a/scripts/install-google-java-format.sh
+++ b/scripts/install-google-java-format.sh
@@ -2,7 +2,7 @@
 
 set -euo pipefail
 
-GOOGLE_JAVA_FORMAT_VERSION="$(grep <"/dependencies/checkstyle/build.gradle" "checkstyle" | awk -F ':' '{print $2}')"
+GOOGLE_JAVA_FORMAT_VERSION="$(grep <"dependencies/google-java-format/build.gradle" "google-java-format" | awk -F ':' '{print $2}')"
 echo "Installing Google Java Format: ${GOOGLE_JAVA_FORMAT_VERSION}"
 
 url=$(curl -s \

--- a/scripts/install-google-java-format.sh
+++ b/scripts/install-google-java-format.sh
@@ -2,6 +2,9 @@
 
 set -euo pipefail
 
+GOOGLE_JAVA_FORMAT_VERSION="$(grep <"/dependencies/checkstyle/build.gradle" "checkstyle" | awk -F ':' '{print $2}')"
+echo "Installing Google Java Format: ${GOOGLE_JAVA_FORMAT_VERSION}"
+
 url=$(curl -s \
   -H "Accept: application/vnd.github+json" \
   -H "Authorization: Bearer $(cat /run/secrets/GITHUB_TOKEN)" \

